assets/xml/archives/icon_item_24_static.xml

```diff
@@ -1,18 +1,21 @@
-        <Blob Name="icon_item_24_static_yar_Blob_002400" Size="0x0900" Offset="0x2400" />
-        <Blob Name="icon_item_24_static_yar_Blob_002D00" Size="0x0900" Offset="0x2D00" />
-        <Blob Name="icon_item_24_static_yar_Blob_003600" Size="0x0900" Offset="0x3600" />
-        <Blob Name="icon_item_24_static_yar_Blob_003F00" Size="0x0900" Offset="0x3F00" />
-        <Blob Name="icon_item_24_static_yar_Blob_004800" Size="0x0900" Offset="0x4800" />
-        <Blob Name="icon_item_24_static_yar_Blob_005100" Size="0x0900" Offset="0x5100" />
-        <Blob Name="icon_item_24_static_yar_Blob_005A00" Size="0x0900" Offset="0x5A00" />
-        <Blob Name="icon_item_24_static_yar_Blob_006300" Size="0x0900" Offset="0x6300" />
-        <Blob Name="icon_item_24_static_yar_Blob_006C00" Size="0x0900" Offset="0x6C00" />
-        <Blob Name="icon_item_24_static_yar_Blob_007500" Size="0x0200" Offset="0x7500" />
+        <Texture Name="gGoldSkulltulaIconTex" OutName="gold_skulltula_icon" Format="rgba32" Width="24" Height="24" Offset="0x0" />
+        <Texture Name="gHeartContainerIconTex" OutName="heart_container_icon" Format="rgba32" Width="24" Height="24" Offset="0x900" />
+        <Texture Name="gPieceOfHeartIconTex" OutName="piece_of_heart_icon" Format="rgba32" Width="24" Height="24" Offset="0x1200" />
+        <!-- Identical to gPieceOfHeartIconTex -->
+        <Texture Name="gPieceOfHeart2IconTex" OutName="piece_of_heart_2_icon" Format="rgba32" Width="24" Height="24" Offset="0x1B00" />
+        <!-- Both identical to gHeartContainerIconTex -->
+        <Texture Name="gHeartContainer2IconTex" OutName="heart_container_2_icon" Format="rgba32" Width="24" Height="24" Offset="0x2400" />
+        <Texture Name="gHeartContainer3IconTex" OutName="heart_container_3_icon" Format="rgba32" Width="24" Height="24" Offset="0x2D00" />
+        <Texture Name="gBossKeyIconTex" OutName="boss_key_icon" Format="rgba32" Width="24" Height="24" Offset="0x3600" />
+        <Texture Name="gCompassIconTex" OutName="compass_icon" Format="rgba32" Width="24" Height="24" Offset="0x3F00" />
+        <Texture Name="gDungeonMapIconTex" OutName="dungeon_map_icon" Format="rgba32" Width="24" Height="24" Offset="0x4800" />
+        <!-- Identical to gGoldSkulltulaIconTex -->
+        <Texture Name="gGoldSkulltula2IconTex" OutName="gold_skulltula_2_icon" Format="rgba32" Width="24" Height="24" Offset="0x5100" />
+        <Texture Name="gSmallKeyIconTex" OutName="small_key_icon" Format="rgba32" Width="24" Height="24" Offset="0x5A00" />
+        <Texture Name="gSmallMagicJarIconTex" OutName="small_magic_jar_icon" Format="rgba32" Width="24" Height="24" Offset="0x6300" />
+        <Texture Name="gBigMagicJarIconTex" OutName="big_magic_jar_icon" Format="rgba32" Width="24" Height="24" Offset="0x6C00" />
+        <Texture Name="gPlayerFaceIconTex" OutName="player_face_icon" Format="rgba16" Width="16" Height="16" Offset="0x7500"/>
```

Comment:
I know, in code, we've been avoiding using "Link" when we mean "Player", because technically the code should work fine for *any* player character. But for this icon, it's a picture of Link's face; even though it's *used* to mark where the player is currently on the map, it's unambiguously a picture of Link. So what do you think about calling it `gLinksFaceIconTex`?

Reply:
Went with `gQuestIconLinkHumanFaceTex` to parallel the `gLinkHumanXXX` assets from object_link_child, thoughts?

Reply:
Yeah, that's fair enough. Even if the other forms don't have dedicated faces, there's nothing to say that they *can't* with enough effort.